frigate/app.py

```diff
@@ -31,48 +35,43 @@
 from frigate.version import VERSION
 from frigate.video import capture_camera, track_camera
 from frigate.watchdog import FrigateWatchdog
+from frigate.types import CameraMetricsTypes
 
 logger = logging.getLogger(__name__)
 
 
 class FrigateApp:
-    def __init__(self):
-        self.stop_event = mp.Event()
-        self.base_config: FrigateConfig = None
-        self.config: FrigateConfig = None
-        self.detection_queue = mp.Queue()
-        self.detectors: Dict[str, EdgeTPUProcess] = {}
-        self.detection_out_events: Dict[str, mp.Event] = {}
-        self.detection_shms: List[mp.shared_memory.SharedMemory] = []
-        self.log_queue = mp.Queue()
-        self.plus_api = (
```

Comment:
Instead of None I decided to "deactivate" this class internally.
The danger is, that caller could try to use methods from this class if a check for None is forgotten.

In addition it is possible in the future to change subscriptions, data or informations during runtime.